infrastructure/output.tf

```diff
@@ -2,6 +2,6 @@ output "worker_ecs_repository_url" {
   value = aws_ecr_repository.worker.repository_url
 }
 
-output "db_accessor_instance_id" {
-  value = aws_instance.db_accessor[0].id
-}
+# output "db_accessor_instance_id" {
+#   value = try(aws_instance.db_accessor[0].id, null)
```

Comment:
@aloftus23 I think we should uncomment this out, could you do that in a separate PR?